### PR TITLE
Bialgebra in the opposite direction

### PR DESCRIPTION
**Pull request by RazinShaikh**

### CODE REVIEW

File: pyzx/editor_actions.py [hunk 1]
```diff
@@ -299,6 +300,86 @@ def bialgebra(g: BaseGraph[VT,ET],
...
+
+    etab: dict = defaultdict(lambda: [0, 0])
+    if edge_type == EdgeType.SIMPLE:
+        etab[upair(new_vertex1, new_vertex2)] = [1, 0]
+    else:
+        etab[upair(new_vertex1, new_vertex2)] = [0, 1]
+    update_etab(etab, new_vertex1, neighbors1, loops1)
+    update_etab(etab, new_vertex2, neighbors2, loops2)
+
+    g.scalar.add_power(-(len(neighbors1)-1)*(len(neighbors2)-1)) #TODO: not sure if this is correct
```
Comment (RazinShaikh) [inline pyzx/editor_actions.py]: I am not completely sure about the global scalar, so I would appreciate if someone else can confirm

Comment (jvdwetering) [reply pyzx/editor_actions.py]: This looks about right to me. It should be the inverse of the scalar in the forward bialgebra rule.

Comment (RazinShaikh) [reply pyzx/editor_actions.py]: Okay, do you know what happens with the self-loops? I am not sure if the self-loop scalar is correct even in the forward case.

**Comment by RazinShaikh** [comment]
The error in the checks seems unrelated to the PR; see https://stackoverflow.com/questions/40183108. Can you run the checks again?

**Comment by RazinShaikh** [comment]
I reran the checks by closing and reopening the PR, and now it passed

File: pyzx/editor_actions.py (hunk 1 above)
Comment (jvdwetering) [reply pyzx/editor_actions.py]: I've never really thought about applying a strong complementarity when there is a self-loop, so I don't know what should happen with the scalar then.

Comment (RazinShaikh) [reply pyzx/editor_actions.py]: I think this is correct so I have removed the todo comment.

**Comment by RazinShaikh** [comment]
this can be merged if it looks good to you @jvdwetering

**Pull request merged**